guava/src/com/google/common/escape/ArrayBasedCharEscaper.java

```diff
@@ -14,11 +14,11 @@
 
 package com.google.common.escape;
 
-import static com.google.common.base.Preconditions.checkNotNull;
-
 import com.google.common.annotations.Beta;
 import com.google.common.annotations.GwtCompatible;
+import org.checkerframework.checker.index.qual.LengthOf;
 import java.util.Map;
+import static com.google.common.base.Preconditions.checkNotNull;
```

Comment:
Sorry, I didn't mean static imports. Imports should be formatted according to https://google.github.io/styleguide/javaguide.html#s3.3-import-statements . I have linked that before, so I assumed that you know.